web_programming/get_top_billioners.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime
+
+import requests
+from rich import box
+from rich import console as rich_console
+from rich import table as rich_table
```

Comment:
```suggestion
from rich import (box, console as rich_console, table as rich_table)
```

Reply:
[isort](https://pypi.org/project/isort/) sorts your imports so you don't have to.  ;-)